Presentation.Web/app/Constants/Constants.ts

```diff
@@ -38,4 +38,8 @@
         static readonly ChooseWhichExcelOptionId = "chooseWhichExcelSelector";
         static readonly ChooseWhichExcelOptionValue = "- - - Vælg hvordan - - -";
     }
+
+    export class UiCustomization {
+        static readonly subtreeIsCompleteHelpText = "Fanenbladdet slås fra hvis alle felter/grupper fjernes, da siden ellers vil være uden indhold.";
```

Comment:
please move this to the other constants container for ui customization... there is already one with texts for that